src/helpers/keboola_helper.js

```diff
@@ -52,9 +53,11 @@ function parseConfigurationOrThrow(configObject) {
             if (!config.actId) throw new Error('Parameter actId is not defined!');
         } else if (config.actionType === ACTION_TYPES.runTask) {
             if (!config.actorTaskId) throw new Error('Parameter actorTaskId is not defined!');
-        } else {
-            if (!config.executionId && !config.token) throw new Error('Parameter token is not defined!');
-            if (!config.executionId && !config.crawlerId) throw new Error('Parameter crawlerId and executionId is not defined!');
+        } else if (config.actionType === ACTION_TYPES.getActorLastRunDatasetItems) {
+            // NOTE: Keboola UI can still use old parameter actId
+            if (!config.actorId && !config.actId) throw new Error('Parameter actorId is not defined!');
```

Comment:
Maybe we could do 

`config.actorId = config.actId || config.actorId;` 

somewhere at the very beginning of the code chain and then everywhere else in the codebase just use the new one. To keep the backward compatibility at the very beginning.